build_data/buildconfig.py

```diff
@@ -0,0 +1,102 @@
+"""
```

Comment:
Why not make the directory (and therefore package name) just be `build` instead of `build_data`. This would then be consistent with the other script specific packages.